src/courses/tests/test_models.py

```diff
@@ -348,6 +348,21 @@ def test_default_object_created(self):
         """ A data migration should make a default block """
         self.assertTrue(Block.objects.filter(block="Default").exists())
 
+    def test_model_protection(self):
+        """ 
+            Quick test to see if Block model deletion is prevented when trying to delete Block model programmatically
+            Block deletion is only prevented when there are CourseStudent models linked via foreign key to the Block model
+        """ 
+        # Setup
+        student = baker.make(User)
+        block = baker.make(Block)
+        course_student = baker.make(CourseStudent, user=student, block=block)
+        self.assertTrue(CourseStudent.objects.count(), 1)
+        self.assertEqual(block, course_student.block)
+
+        # see if models.PROTECT is in place
+        self.assertRaises(Exception, block.delete)
```

Comment:
Be more specific in which exact exception you are testing for.  Otherwise there could be a different exception that occurs  do to some other error and your test will pass when it shouldn't.